### PR TITLE
Bob the builder, can we fix it ?

### PR DESCRIPTION
**Pull request by zaclegarssure**
Bob the builder, yes we can !

### CODE REVIEW

**Comment by zaclegarssure** [comment]
actually small notif update can you have a look @TheblueMan003 ?

**Comment by zaclegarssure** [comment]
ah wait my bad you already store it before if it does not exists

**Comment by zaclegarssure** [comment]
well the behavior should be the same I guess

File: app/src/main/java/com/github/palFinderTeam/palfinder/notification/NotificationService.kt [hunk 1]
```diff
@@ -100,22 +98,23 @@ class NotificationService @Inject constructor(
...
+                        val news = meetup.participantsId.subtract(meta.participant.toSet())
+                            .filter { it != id }
+                        meta.participant.addAll(news)
+                        meetupsMetaData.store(m, meta)
                         if (news.isNotEmpty()) {
                             val names = profileService.fetch(news.toList())!!.map { it.name }
                                 .reduce { x, y -> "$x, $y" }
-                            data.participant.addAll(news)
-                            meetupsMetaData.store(m, data)
+                            meta.participant.addAll(news)
```
Comment (TheblueMan003) [inline app/src/main/java/com/github/palFinderTeam/palfinder/notification/NotificationService.kt:109]: Storing the meta twice

**Review by TheblueMan003 — APPROVED** [review]
Otherwise seems to do the same thing

File: app/src/main/java/com/github/palFinderTeam/palfinder/notification/NotificationService.kt (hunk 1 above)
Comment (zaclegarssure) [reply app/src/main/java/com/github/palFinderTeam/palfinder/notification/NotificationService.kt:109]: very meta

**Review by TruddyTheDuddi — APPROVED** [review]
ezgif com-gif-maker(9)

**Pull request merged**